frontend/src/views/DiagnosticEditor/index.vue

```diff
@@ -39,18 +40,24 @@
             </v-col>
 
             <div v-if="isTeledeclarationYear">
-              <p v-if="!hasActiveTeledeclaration && !canSubmitTeledeclaration" class="text-caption ma-0 pl-4">
+              <p
+                v-if="!hasActiveTeledeclaration && !canSubmitTeledeclaration && showApproPanel"
+                class="text-caption ma-0 pl-4"
+              >
                 <v-icon small>mdi-information</v-icon>
                 Vous pourrez télédéclarer ce diagnostic après avoir remplir les données d'approvisionnement
               </p>
-              <p v-else-if="!hasActiveTeledeclaration" class="text-body-2 pl-4 mb-0 d-md-flex align-center">
+              <p
+                v-else-if="!hasActiveTeledeclaration && showApproPanel"
```

Comment:
Why not display this prompt if the appro panel isn't shown ? Should there be a check on `canSubmitTeledeclaration` also ?